chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/onetime/criteriaremoval/RemoveCriteriaJob.java

```diff
@@ -30,18 +48,23 @@ public RemoveCriteriaJob() throws Exception {
     public void execute(JobExecutionContext jobContext) throws JobExecutionException {
         SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
         LOGGER.info("********* Starting the Remove Criteria job. *********");
-        for (String criteria : CRITERIA_TO_REMOVE) {
-            try {
-                String[] criterionElements = criteria.split(":");
-                CertificationCriterionDTO certDto = certCriteriaDao.getByNumberAndTitle(criterionElements[0], criterionElements[1]);
-                certDto.setRemoved(true);
-                certCriteriaDao.update(certDto);
-                LOGGER.info("Updated criteria " + criteria);
-            } catch (final Exception ex) {
-                LOGGER.error("Exception updating criteria " + criteria, ex);
+
+        if (ff4j.check(FeatureList.ERD_PHASE_2)) {
+            for (String criteria : CRITERIA_TO_REMOVE) {
+                try {
+                    String[] criterionElements = criteria.split(":");
+                    CertificationCriterionDTO certDto = certCriteriaDao.getByNumberAndTitle(criterionElements[0], criterionElements[1]);
+                    certDto.setRemoved(true);
+                    certCriteriaDao.update(certDto);
+                    LOGGER.info("Updated criteria " + criteria);
+                } catch (final Exception ex) {
+                    LOGGER.error("Exception updating criteria " + criteria, ex);
+                }
             }
+            CacheManager.getInstance().clearAll();
```

Comment:
Can you also clear the shared store here? I suspect this job will be run after the shared store is deployed.

Reply:
Can I? Or does that depend on OCD-3949 going to STG? Also not sure if that branch even _has_ a "clear all store" method

Reply:
Don't know! Question for @tmy1313 . I believe it needs it at some point anyway so I want to leave this comment here but approve the feature going forward if the clearing code is not available yet. If it depends on OCD-3949 then you'll need to make a commit with that clearing code when this goes to QA I guess.

Reply:
My thought would be to add a checkbox to OCD-3949 that indicates that we need to clear the Shared Listing Store when this job is run and then implement the change when this ticket goes to STG.  I would leave the code for this ticket alone for now.  But this does leave implications for testing this change - how do we test this change in any environment where OCD-3949 has already been implemented?  We probably need to discuss...